specs/sb/api.md

```diff
@@ -573,9 +573,9 @@ existing system.
 </thead>
 <tbody>
 <tr>
-  <td>organization_guid*</td>
+  <td>context_guid*</td>
   <td>string</td>
-  <td>The Cloud Controller GUID of the organization under which the service is to be provisioned. Although most brokers will not use this field, it could be helpful in determining data placement or applying custom business rules.</td>
+  <td>The Cloud Controller GUID of the context under which the service is to be provisioned. Although most brokers will not use this field, it could be helpful in determining data placement or applying custom business rules.</td>
```

Comment:
@shalako can you provide me with a good/real example that I can put there? I might be able to come up with one that's totally made-up but I'd prefer to include one that's based on a real implementation.

Would a SB use this during the billing cycle to tell a platform how much to charge the owner of each "context" ?